### PR TITLE
Use wrapper functions to fix new retail APIs

### PR DESCRIPTION
**Pull request by ado21**
Hey, here is reworked code with wrappers, if the other solution is better simply close this PR:

Creation of 3 wrapper functions in GladiusEx.lua
Calling new wrapper functions from all places where necessary
Fixing spell id for scatter shot used in testing mode

Havent seen any bug so far with this.

### CODE REVIEW

File: modules/classicon.lua [hunk 1]
```diff
@@ -53,7 +53,7 @@ function ClassIcon:OnEnable()
 	end
 	
 	self:InsertTestDebuff(8122, 8, "Magic") -- Psychic Scream
-	self:InsertTestDebuff(19503, 4, nil) -- Scatter Shot
```
Comment (vendethiel) [inline modules/classicon.lua:56]: this doesnt exist in cata, just remove the line

Comment (ado21) [reply modules/classicon.lua:56]: gone

**Comment by vendethiel** [comment]
Sorry, looks like #104 is in a slightly better shape.
I merged your PR for LibCooldownTracker, thanks!
And in general, thank you very much for your contribution :).

**Comment by ado21** [comment]
Sure, no problem, as long as addon is being updated im satisfied :D

**Pull request closed without merging**